ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -67,7 +66,7 @@
   private final int maxTransactionRetentionHours;
   private final Clock clock;
 
-  private final Queue<Hash> newPooledHashes;
+  private final EvictingQueue<Hash> newPooledHashes;
```

Comment:
Why do we need to specialize this type to EveictingQueue?  Isn't `iterator` available in Queue via the Collection supertype?

Reply:
I was just making it more specific since I was here anyways. Return specific, accept general and all that.